### ISSUE TITLE
Can your code be used for image segmentation?

### ISSUE THREAD

**Issue by Yang-Code984** [SANCHES-Pedro/Diff-SCM#1]
If it can be used for segmentation, could you share the format of the training data set and the format of the test set
